remappings.txt

```diff
@@ -1,6 +1,7 @@
 forge-std/=lib/forge-std/src/
 solmate/=lib/solmate/
-@openzeppelin/=lib/openzeppelin-contracts
+@openzeppelin/contracts/=lib/openzeppelin-contracts-upgradeable/lib/openzeppelin-contracts/contracts/
```

Comment:
TODO: delete the @openzeppelin-contracts forge dependency, since we're now using the same version installed by the upgradeable contracts